cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaClusterCreator.java

```diff
@@ -58,6 +59,7 @@ public class KafkaClusterCreator {
     private boolean scaleDownCheckFailed = false;
     private boolean usedToBeBrokersCheckFailed = false;
     private final List<Condition> warningConditions = new ArrayList<>();
+    private Set<Integer> scalingDownBlockedNodes = new LinkedHashSet<>();
```

Comment:
Why do you need LinkedHashSet? Might be worth adding a comment.

Reply:
Good spot. No it's not actually needed. The order doesn't matter here.